### ISSUE TITLE
Class field documentation and formatting incorrect

### ISSUE THREAD

**Issue by cmann1** [godotengine/godot-vscode-plugin#373]
### Godot version

3.4.stable

### VS Code version

1.67.2

### Godot Tools VS Code extension version

1.3.0

### System information

Windows 7

### Issue description

![image](https://user-images.githubusercontent.com/4889816/171270321-d4544c39-0012-4a9a-86ab-a0c8e35216ce.png)

This is sort of two related issues.
First issue is that the documentation for all preceding variables will be merged with the current one if there aren't spaces between them.
The second is that a double hash (`##`) causes the text to become very large - I'm guessing it's being interpreted as a markdown heading.

### Steps to reproduce

Create a gd file with this content:
```gdscript
class_name test_motion

# First comment - formatting should be fine, and won't be included in documentation below.
export var first: Array

## Second comment - font should be very large.
export var second: float = 0
## Third comment - hovering over `third` should show this, as well as the comment for `second`
export var third: float
```

**Comment by atirut-w** [comment]
The documentations are generated by Godot's LSP, so you should transfer this to Godot's repository instead.

The second problem is already fixed on the master branch there, but I am not sure about the first one.

**Comment by DaelonSuzuka** [comment]
A tentative fix for this issue will be in #488

![image](https://github.com/godotengine/godot-vscode-plugin/assets/18042232/1c69dfc4-fc05-4262-bbfc-83b49beefdd4)
